openedx/features/teams/views.py

```diff
@@ -0,0 +1,267 @@
+    leave_team_url = reverse('team_membership_detail', args=[team_id, request.user.username])
+
+    team_administrator = (has_access(request.user, 'staff', course_key)
+                          or has_discussion_privileges(request.user, course_key))
+
+    is_member_of_any_team = CourseTeamMembership.user_in_team_for_course(request.user, course_key)
+
+    is_user_member_of_this_team = bool(CourseTeamMembership.objects.filter(team=team, user=user).first())
+
+    context = {
+        'course': course,
+        'user_has_team': is_member_of_any_team,
+        'is_team_full': course.teams_max_size <= len(team.users.all()),
+        'is_user_member_of_this_team': is_user_member_of_this_team,
+        'room_url': embed_url,
+        'join_team_url': reverse('team_membership_list'),
+        'team': team,
+        'team_administrator': team_administrator,
+        'leave_team_url': leave_team_url,
+        'country': str(countries.countries[team.country]),
+        'language': dict(settings.ALL_LANGUAGES)[team.language],
+    }
+
+    return render_to_response("teams/view_team.html", context)
+
+
+@login_required
+def update_team(request, course_id, team_id):
+    course_key = CourseKey.from_string(course_id)
+    course = get_course_with_access(request.user, "load", course_key)
+
+    team_administrator = (has_access(request.user, 'staff', course_key)
+                          or has_discussion_privileges(request.user, course_key))
+    if not team_administrator:
+        raise Http404
+
+    try:
+        team = CourseTeam.objects.get(team_id=team_id)
+    except CourseTeam.DoesNotExist:
+        raise Http404
+
+    context = {
+        'course': course,
+        'team': team,
+        'countries': list(countries),
+        'languages': [[lang[0], _(lang[1])] for lang in settings.ALL_LANGUAGES],
+        'user_has_privilege': team_administrator,
+        'template_view': 'update'
+    }
+
+    return render_to_response("teams/create_update_team.html", context)
+
+
+@login_required
+def edit_team_memberships(request, course_id, team_id):
+    course_key = CourseKey.from_string(course_id)
+    course = get_course_with_access(request.user, "load", course_key)
+
+    team_administrator = (has_access(request.user, 'staff', course_key)
+                          or has_discussion_privileges(request.user, course_key))
+    if not team_administrator :
+        raise Http404
+
+    try:
+        team = CourseTeam.objects.get(team_id=team_id)
+    except CourseTeam.DoesNotExist:
+        raise Http404
+
+    team_data = serialize(
+        team,
+        request,
+        CustomCourseTeamSerializer,
+        {'expand': ('user',)},
+        many=False
+    )
+
+    context = {
+        'course': course,
+        'members': team_data['membership'],
+        'team_id': team_id
+    }
+
+    return render_to_response("teams/edit_memberships.html", context)
+
+
+def get_alphabetical_topics(course_module):
```

Comment:
let's verify if we've not made changes in the code then let's just import instead of copying the code.